scripts/rook.sh

```diff
@@ -8,6 +8,22 @@ ROOK_CEPH_CLUSTER_VERSION="v14.2.10"
 KUBECTL_RETRY=5
 KUBECTL_RETRY_DELAY=10
 
+trap log_errors ERR
+
+# log_errors is called on exit (see 'trap' above) and tries to provide
+# sufficient information to debug deployment problems
+function log_errors() {
+	# enable verbose execution
+	set -x
+	kubectl get nodes
+	kubectl -n rook-ceph get events
+	kubectl -n rook-ceph describe pods
+	kubectl -n rook-ceph logs -l app=rook-ceph-operator
+	kubectl -n rook-ceph get CephClusters -oyaml
+	kubectl -n rook-ceph get CephFilesystems -oyaml
+	kubectl -n rook-ceph get CephBlockPools -oyaml
```

Comment:
One last thing I guess, don't you want to `exit 1` after showing the logs so the CI fails?

Reply:
The script runs with `bash -e` so when an error is caught, the script exits with the error value. An additional `exit 1` is not needed (also tested locally with a small script).